src/commands/channelAssign.ts

```diff
@@ -0,0 +1,15 @@
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { PermissionFlagsBits } from 'discord-api-types/v9';
+import { CommandInteraction } from 'discord.js';
+import { addChannel } from '../buttons';
+
+export const data = new SlashCommandBuilder()
+    .setName('channelmanagement')
+    .setDescription('Setup user channel management')
+    .setDefaultMemberPermissions(PermissionFlagsBits.Administrator);
```

Comment:
Currently I don't think that this would made this command usable since normal users wouldn't be able to use it. I am not sure how to deal with this. One thing that comes to my mind is to check if user has `verified` role. This will means that everyone will be able to invoke this command but when there will be check at start they will not see possible channels

Reply:
Resolved off chat